src/componentfuncs/Comments.jsx

```diff
@@ -32,9 +32,11 @@ export function LeaveComment (props) {
   const showing = props.show
   const currentId = props.id
 
-  const commentValidate = (comment) => {
+  const formValidation = (comment) => {
     if (typeof comment === 'string' && comment.length > 15){
-      postComment(currentId, comment)
+      postComment(currentId, comment).catch((err) => {
+        setValid('Network Error, Try Again')
```

Comment:
Although this is working you are changing the data type of validity here. Its initial state is a boolean of true and you set it's state to false elsewhere. Changing it to a string is is not good practice. I would strongly suggest you find another way to get the error message rendered on screen.